src/background.js

```diff
@@ -33,6 +34,7 @@ function updateButtonForTab(tabId, changeInfo) {
 
 function init() {
   log('init()');
+  Metrics.init(window.BroadcastChannel, browser.tabs);
```

Comment:
So, I'm not really sure why we're passing in the BroadcastChannel here.  I suspect it's the right thing to do, but could you elaborate a little more on it (maybe in a comment in `metrics.js`)?